operator/config/watcher/patches/watcher_service_patch.yaml

```diff
@@ -0,0 +1,3 @@
+- op: replace
+  path: /spec/serviceInfo/name
+  value: module-manager-event-service
```

Comment:
this replace only works with default kustomize, with namePrefix `module-manager-`, it's not fit for KCP and load test deployment with namePrefix `kmm-`.
I think we should align the namePrefix, otherwise patch the name differently in each deployment case.